### ISSUE TITLE
long wait on startup if soundcards change

### ISSUE THREAD

**Issue by Holzhaus** [Holzhaus/mixxx-gh-issue-migration#273]
Reported by: @kousu
Date: 2009-04-26T04:25:46Z
Status: Won't Fix
Importance: Low
Launchpad Issue: [lp367132](https://bugs.launchpad.net/bugs/367132)

---

If I plug in my external soundcard, close mixxx, pull the soundcard, and restart there is a long wait the next time mixxx starts.

Mixxx1.7-beta1, ubuntu 8.10.

**Comment by Holzhaus** [comment]
Commented by: @rryan
Date: 2010-09-11T08:37:52Z

---

This still happens to me even with portaudio trunk and the 1.8 release branch.

**Comment by Holzhaus** [comment]
Commented by: @bkgood
Date: 2011-01-02T01:34:57Z

---

Can anyone repro this with the 1.9 branch? Maybe the audio routing rewrite solved this.

**Comment by Holzhaus** [comment]
Commented by: @rryan
Date: 2011-12-07T23:04:01Z

---

Still happens to me every now and then. PA devs have said it's likely a dmix screwup

**Comment by Holzhaus** [comment]
Issue closed with status "Won't Fix".
